src/interfaces/iTransactionRelationship.ts

```diff
@@ -12,4 +12,5 @@ export interface TransactionRelationship {
   lat?: string;
   long?: string;
   TxSts?: string;
+  tenantId: string; // Always required - no longer optional
```

Comment:
This interface follows the ISO20022 standard, so the case should be `TenantId`
```suggestion
  TenantId: string;
```